src/plugins/newsfeed/public/lib/api.test.mocks.ts

```diff
@@ -18,3 +19,9 @@ export const driverInstanceMock = driverMock.create();
 jest.doMock('./driver', () => ({
   NewsfeedApiDriver: jest.fn().mockImplementation(() => driverInstanceMock),
 }));
+
+jest.doMock('./never_fetch_driver', () => ({
+  NeverFetchNewsfeedApiDriver: jest
+    .fn()
+    .mockImplementation(() => new NeverFetchNewsfeedApiDriver()),
```

Comment:
```suggestion
  NeverFetchNewsfeedApiDriver: jest.fn(() => new NeverFetchNewsfeedApiDriver()),
```

that shorthand makes the code look tidy